Authorization/Authorization.Presentation/DependencyInjection/PresentationDependencyInjection.cs

```diff
@@ -0,0 +1,151 @@
+                var operationalDbContext = scope.ServiceProvider.GetRequiredService<PersistedGrantDbContext>();
+                operationalDbContext.Database.Migrate();
+            }
+
+            return services;
+        }
+
+        private static void AddClient(ConfigurationDbContext context, IConfiguration configuration)
+        {
+            var clients = new List<Client>()
+            {
+                GetClient(configuration, "ServicesApi"),
+                GetClient(configuration, "OfficesApi")
+            };
+
+            foreach (var client in clients)
+            {
+                if (!context.Clients.Any(c => c.ClientId == client.ClientId))
+                {
+                    context.Clients.Add(client.ToEntity());
+                    Console.WriteLine($"Added Client: {client.ClientName}");
+                }
+            }
+        }
+
+        private static Client GetClient(IConfiguration configuration, string clientName)
+        {
+            return new Client
+            {
+                ClientId = configuration.GetSection("AuthorizationClients").GetSection(clientName)["ClientId"]!,
+                ClientName = configuration.GetSection("AuthorizationClients").GetSection(clientName)["ClientName"],
+                ClientSecrets = { new Secret(configuration.GetSection("AuthorizationClients").GetSection(clientName)["ClientSecret"].Sha256()) },
+                AllowedGrantTypes = GrantTypes.Code,
+                RedirectUris = { $"{configuration.GetSection("AuthorizationClients").GetSection(clientName)["ClientBaseUrl"]}signin-oidc" },
+                PostLogoutRedirectUris = { $"{configuration.GetSection("AuthorizationClients").GetSection(clientName)["ClientBaseUrl"]}signout-callback-oidc" },
+                AllowedScopes =
+                {
+                    "openid",
+                    "profile",
+                    "email",
+                    "roles",
+                    "api_profile",
+                    "api_email",
+                    "api_roles"
+                },
+                AlwaysIncludeUserClaimsInIdToken = true,
+                AlwaysSendClientClaims = true
+            };
+        }
+
+
+        private static void AddIdentityResources(ConfigurationDbContext context)
+        {
+            var identityResources = new List<IdentityResource>
+            {
+                new IdentityResources.OpenId(),
+                new IdentityResources.Profile(),
+                new IdentityResource("email", new[] { JwtClaimTypes.Email }),
+                new IdentityResource("roles", new[] { JwtClaimTypes.Role })
+            };
+
+            foreach (var resource in identityResources)
+            {
+                if (!context.IdentityResources.Any(ir => ir.Name == resource.Name))
+                {
+                    context.IdentityResources.Add(resource.ToEntity());
+                    Console.WriteLine($"Added Identity Resource: {resource.Name}");
```

Comment:
Either change it to writing to logs or remove it

Reply:
Fixed